.github/workflows/publish.yml

```diff
@@ -39,17 +41,27 @@ jobs:
             package: ${{ inputs.package }}
             branch: ${{ inputs.branch }}
 
+    integration-tests:
+        if: ${{ inputs.package != 'dbt-adapters' && inputs.package != 'dbt-tests-adapter' }}
+        uses: ./.github/workflows/_integration-tests.yml
+        with:
+            package: ${{ inputs.package }}
+            branch: ${{ inputs.branch }}
+            repository: ${{ github.repository }}
+        secrets: inherit
+
     generate-changelog:
-        needs: unit-tests
+        needs: [unit-tests, integration-tests]
```

Comment:
will this pass on skip by if or fail. I can't remember the boolean logic from me doing similar a year ago. You may need an "Always" clause or something like that.

Reply:
Oh you might be right. I'm hesitant to put that in this PR though. For some reason I feel like it should be a separate PR since it's not directly related to migrating dbt-athena into this repo. Though I suppose it's also related to adding integration tests here. I'm also trying to minimize how many commits are in this PR since it won't be a squash merge. Are you comfortable with waiting?

As a side note, I believe what we want is `if: !failure()`.

Reply:
Yeah, I can absolutely buy that. I just wanted to make sure we at least flag it and know that it's something we need to do.